### ISSUE TITLE
Sequoia checker

### ISSUE THREAD

**Issue by staze** [laurentpertois/Sonoma-Compatibility-Checker#2]
Looks like you should largely be able to re-use Sonoma checker code, just removing the 2018 and 2019 versions of macbook air...

**Comment by islamadel** [comment]
vote

According to this list (https://github.com/dylanaraps/neofetch/pull/1944/files) the min. value for macbook air would be then 91

`elif [[ "$MODELNAME" == "MacBookAir" && "$MODELVERSION" -ge 91 && "$MEMORYINSTALLED" -ge "$MINIMUMRAM" && "$FREESPACE" -ge "$MINIMUMSPACE" ]]; then
	    COMPATIBILITY="True"`

**Comment by staze** [comment]
yup, that's what I was just what I was going to post. =) you could probably say modelversion -ge 90, since the 2019 was an 82, and 2018 was 81.
